src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
+    tempValue.textContent = currentTemperature;
+    tempColor();
+};
+
+const addDegree = () => {
+    currentTemperature += 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const subtractDegree = () => {
+    currentTemperature -= 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
```

Comment:
Notice that the difference between these functions is `+=` and `-=`.

Consider combining this logic into one helper function `handleTempChange` that could be registered with both the ⬆️ and ⬇️ buttons.

```javascript
const handleTempChange = (event) => {
    if (event.target.id === 'increaseTempControl') {
        currentTemperature ++;
    } else {
       currentTemperature--;
    }
    tempValue.textContent = currentTemperature;
    temperatureDisplay();
};